content/integrations/dyn.md

```diff
@@ -34,14 +33,15 @@ Once you have created a Datadog Read only user:
 4. Configure your [Dyn integration](https://app.datadoghq.com/account/settings#integrations/dyn) inside your Datadog application.
 {{< img src="integrations/dyn/dyn_integration.png" alt="Dyn Integration" style="width:75%;" >}}
 
-5. Select your zones you want to monitor:
-{{< img src="integrations/dyn/dyn_zone.png" alt="Dyn zone" style="width:50%;" >}}
-
+5. Select your zones you wish to monitor:<br>
```

Comment:
Same feedback as last time "Select the zones you wish to monitor"

Reply:
👍